graph-http/src/traits/link.rs

```diff
@@ -28,3 +28,18 @@ impl ODataLink for serde_json::Value {
         self["@odata.deltaLink"].as_str().map(|s| s.to_string())
     }
 }
+
+pub trait ODataNextLink<V, RHS = Self> {
+    fn next_link(&self) -> Option<String>;
+    fn value(&mut self) -> &mut Vec<V>;
+}
+
+impl ODataNextLink<serde_json::Value> for serde_json::Value {
+    fn next_link(&self) -> Option<String> {
+        self["@odata.nextLink"].as_str().map(|s| s.to_string())
+    }
+
+    fn value(&mut self) -> &mut Vec<serde_json::Value> {
+        self["@odata.value"].as_array_mut().unwrap() // todo: can we replace this unwrap with something else ?
```

Comment:
You should either map this to an Option or return it as a Result. We don't know if this value will be there and we don't want to unwrap without a very good reason. I usually only ever do so when dealing with threads because that is common practice in Rust.

Reply:
Yeah, good idea. 
It will solve the unwrap and will be much easier to implement.